src/java/fr/paris/lutece/plugins/appointment/service/EntryService.java

```diff
@@ -570,6 +573,49 @@ public static void getHtmlEntry( Map<String, Object> model, int nIdEntry, String
         stringBuffer.append( template.getHtml( ) );
     }
 
+    /**
+     * Get the html part of the additional entry of the form
+     * 
+     * @param nIdEntry
+     *            the entry id
+     * @param stringBuffer
+     *            the string buffer
+     * @param locale
+     * @param bDisplayFront
+     * @param request
+     * @param codesPredemandeList
+     */
+    public static void getSpecificHtmlEntry( Map<String, Object> model, Entry entry, StringBuilder stringBuffer, Locale locale, boolean bDisplayFront,
```

Comment:
Can you update the comments so we know exactly what this method does, compared to "getHtmlEntry"